apps/artist/templates/page.jade

```diff
@@ -17,7 +17,7 @@ block content
         | #{artist.get('forsale_artworks_count')} for sale
       #artist-page-sort
         label.garamond-select: select( dir="rtl" )
-          for val, label in { "Sort by relevance": '', "Sort by recently added": '-published_at' }
+          for val, label in { "Recently Updated": "-partner_updated_at", "Recently added": '-published_at', "Artwork year - asc": "year" , "Artwork year - desc": '-year' }
```

Comment:
We show `asc` and `desc` on Force currently:

<img width="285" alt="screen shot 2017-01-27 at 1 50 23 pm" src="https://cloud.githubusercontent.com/assets/1457859/22383388/944924a8-e497-11e6-85fc-4895bebb7cc9.png">